### ISSUE TITLE
GTX 670 Low Hash Rate

### ISSUE THREAD

**Issue by BCwhoLikesPHP** [xmrig/xmrig-nvidia#95]
I am looking for a bit of help here.

I have a GTX 670 2GB running on Windows 10 64bit, I installed the Cuda 9.1 Developer Packackage. 

For Some reason, the hashrate is a low 140h/s but many benchmark sites claim that the card is capable of hashing in the mid to high 300's . Any idea of what I need to do to get hashing at the higher rates? I have access to a lot of these cards and if I can get it to work out, it would be good.

Any sample config file setting would help a great deal. Thanks!

**Comment by Mek7** [comment]
I have a laptop with 670M and am getting only around 140 h/s as well.

**Comment by Hero9909** [comment]
Seems to applay to cards only running in cli mode (without desktop)

**Comment by Mek7** [comment]
Not really, I was running desktop Windows. But I bought a new PC since then so this issue does not apply to me anymore.
